packages/running/src/index.tsx

```diff
@@ -559,6 +560,12 @@ class Section extends PanelWithToolbar {
         ? 'tree'
         : 'list';
     this.addWidget(this._listWidget);
+
+    if (this._manager.toolbarButtons) {
+      this._manager.toolbarButtons.forEach(button =>
+        this.toolbar.addItem(button.id, button)
```

Comment:
`button.id` is empty for command toolbar buttons. We may need to change that or, the API.

Reply:
Solved in https://github.com/jupyterlab/jupyterlab/pull/16341/commits/88ba6852382b197656b04ef753bbb404d049c33f